barman/postgres.py

```diff
@@ -665,6 +665,43 @@ def has_checkpoint_privileges(self):
                 )
                 return None
 
+    @property
+    def has_monitoring_privileges(self):
+        """
+        Returns true if the current user is a superuser or if the user has the
```

Comment:
```suggestion
        Returns ``True`` if the current user is a superuser or if the user has the
```

(Sphinx recommendation for literal values).